resources/assets/js/snipeit.js

```diff
@@ -212,24 +212,24 @@ $(document).ready(function () {
                     };
                     return data;
                 },
-                processResults: function (data, params) {
+                /* processResults: function (data, params) {
```

Comment:
Should we just remove commented out code?

Reply:
Maybe eventually? I wanted to do this kinda 'safe' just to be extra-cautious. But, we do have version control - maybe I'm being overcautious.